src/components/XdrTypeSelect/index.tsx

```diff
@@ -102,6 +102,16 @@ export const XdrTypeSelect = ({ error }: XdrTypeSelectProps) => {
               <OptionItem key={`popular-${p}`} option={p} />
             ),
           )}
+          {guessedTypes ? (
+            <>
+              <OptionItem sectionTitle="Possible Types" />
+              {guessedTypes.map((o) => (
+                <>
+                  <OptionItem key={`guessed-${o}`} option={o} />
+                </>
```

Comment:
I don't think we need to wrap it in a fragment here.

```suggestion
                <OptionItem key={`guessed-${o}`} option={o} />
```